# remap_pwm.yml
# This file defines the mapping used in map_thrust.py
# to convert the output of the real Maelstrom control stack
# to the input to the UUV simulator thruster plugin.

real_topic: /command/pwm
sim_topic_prefix: /maelstrom/thrusters/
sim_topic_suffix: /input
thruster_mapping:
# - [sim ID, real field in pwm msg]
  - [HSA, heave_stbd_aft]
  - [HPA, heave_port_aft]
  - [HSF, heave_stbd_fwd]
  - [HPF, heave_port_fwd]
  - [SSL, surge_stbd_lo]
  - [SPL, surge_port_lo]
  - [SWF, sway_fwd]
  - [SWA, sway_aft]